src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw.module.ts

```diff
@@ -183,7 +188,50 @@ const routes: Routes = [
   },
   {
     path: 'multisite',
+    canActivate: [FeatureTogglesGuardService, ModuleStatusGuardService],
+    data: {
+      moduleStatusGuardConfig: {
+        uiApiPath: 'rgw/multisite',
+        redirectTo: 'rgw/error',
+        header: 'Multi-site not configured',
+        button_name: 'Add Multi-site Configuration',
+        button_route: '/rgw/multisite/create',
+        button_title: 'Add multi-site configuration (realms/zonegroups/zones)',
+        secondary_button_name: 'Import Multi-site Configuration',
+        secondary_button_route: 'rgw/multisite/import',
+        secondary_button_title:
+          'Import multi-site configuration (import realm token from a secondary cluster)'
+      },
```

Comment:
this is something which is being removed through: https://github.com/ceph/ceph/pull/54247/files